website/content/en/docs/ansible/quickstart.md

```diff
@@ -376,4 +376,4 @@ For more information, refer [cli][addcli] doc.
 [docker-tool]:https://docs.docker.com/install/
 [kubectl-tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [addcli]: /docs/cli/operator-sdk_add_api
-[olm-user-guide]: /docs/olm-integration/user-guide
+[quickstart-bundle]: /docs/olm-integration/legacy/quickstart-bundle
```

Comment:
Add a TODO to update this link once the ansible plugin for KB project structure exists?

/cc @asmacdo